README.md

```diff
@@ -16,15 +16,15 @@ The VM currently supports the following hardfork rules:
 - `Byzantium`
 - `Constantinople`
 - `Petersburg` (default)
-- `Istanbul` (`beta`)
+- `Istanbul`
 
 If you are still looking for a [Spurious Dragon](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-607.md) compatible version of this library install the latest of the `2.2.x` series (see [Changelog](./CHANGELOG.md)).
 
 ##### Istanbul Harfork Support
 
-A feature-complete `Istanbul` HF implementation is available since the `v4.1.0`
-VM release. You can activate an `Istanbul` VM by using the `istanbul`
-`hardfork` option flag.
+An Ethereum test suite compliant `Istanbul` HF implementation is available
+since the `v4.1.1` VM release. You can activate an `Istanbul` VM by using the
+`istanbul` `hardfork` option flag.
```

Comment:
When do you think we should make it the default HF? After the HF happens, right?

Reply:
Thought about this during preparation as well. Yes, probably (after HF). This will be a breaking change and would require a major release. We might also want to look for another feedback round and see for another 2-4 weeks how this release pass the reality test 😄, generally confident though.

Reply:
A major release for the change to the default HF makes sense. I wonder if we should keep those major releases small (i.e. mostly changing the default), or use those to also introduce other changes. I haven't thought enough about it though.